src/quantum/perfherder.jsx

```diff
@@ -1,283 +0,0 @@
-
-        // if (!idx && this.props.unit) {
-        //   label += this.props.unit;
-        // }
-        return (
-          <g
-            className={cx('tick', 'tick-y', {
-              'tick-axis': idx === 0,
-              'tick-secondary': idx > 0,
-            })}
-            key={`tick-${tick}`}
-          >
-            <line x1={0} y1={y} x2={width} y2={y} />
-            <text x={2} y={y}>
-              {label}
-            </text>
-          </g>
-        );
-      });
-      const yFormat = timeFormat('%b');
-      const $xAxis = xScale.ticks(timeMonth.every(1)).map((tick) => {
-        const x = xScale(tick);
-        const label = yFormat(tick);
-
-        return (
-          <g className={cx('tick', 'tick-x')} key={`tick-${tick}`}>
-            <text x={x} y={height - 5}>
-              {label}
-            </text>
-          </g>
-        );
-      });
-      const $legend = signatureLabels.length > 1
-        ? signatureLabels.map((label, idx) => (
-          <text
-            className={`legend series-${idx}`}
-            x={27 + 60 * idx}
-            y={15}
-            key={`legend-${label}`}
-          >
-            {label}
-          </text>
-        ))
-        : null;
-      const $reference = referenceX ? (
-        <line
-          key="reference-y"
-          className="reference reference-y"
-          x1={referenceX}
-          y1={min(referenceYs)}
-          x2={referenceX}
-          y2={height - 25}
-        />
-      ) : null;
-
-      svg = (
-        <svg height={height} width={width}>
-          {$yAxis}
-          {$xAxis}
-          {$evolutions}
-          {$reference}
-          {$legend}
-        </svg>
-      );
-    } else {
-      svg = 'Loading Perfherder …';
-    }
-
-    const link = URL({
-      path: 'https://treeherder.mozilla.org/perf.html#/graphs',
-      query: {
-        timerange: 7776000,
-        series: (signatures
-          ? [...signatures.keys()]
-          : Object.values(this.props.signatures)
-        ).map(signature => ['mozilla-central', signature, 1, framework]),
-      },
-    });
-
-    return (
-      <Widget
-        key={this.props.title}
-        {...this.props}
-        link={link}
-        target={
-          this.props.target != null ? this.props.target : 'No regressions'
-        }
-        className="graphic-widget graphic-timeline"
-        content={svg}
-        loading={!evolutions}
-        explainer={`${explainer} (14-day moving median, variance band between 1st/3rd quantile)`}
-        viewport={size => (this.viewport = size)}
-      />
-    );
-  }
-}
-
-PerfherderWidget.defaultProps = {
-  signatures: '',
-  framework: 1,
-  reference: '',
-};
-PerfherderWidget.propTypes = {
-  title: PropTypes.string.isRequired,
-  signatures: PropTypes.object,
-  framework: PropTypes.number,
-  explainer: PropTypes.string,
-  reference: PropTypes.string,
-  target: PropTypes.string,
-};
-
-export default withErrorBoundary(PerfherderWidget);
```

Comment:
@davehunt  We no longer require `perfherder.jsx`  this is just one step of many in an attempt to reduce the total download size of this dashboard:  https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/251